client/components/marked-lines/style.scss

```diff
@@ -0,0 +1,46 @@
+.marked-lines {
+	padding: 0;
+	background-color: #e6ecf0;
+	font-family: $monospace;
+	overflow-x: scroll;
+	white-space: pre;
+}
+
+.marked-lines__line {
+
+}
+
+.marked-lines__marked-line {
+	background-color: #e0c5c9;
+
+	& .marked-lines__line-number {
+		background-color: #d4bbc1;
+	}
+}
+
+.marked-lines__line-number {
+	display: inline-block;
+	margin-right: 4px;
+	padding: 4px;
+	background-color: #d6e0e8;
+	color: #537794;
+
+	&:first-child {
+		padding-top: 4px;
+	}
+
+	&:last-child {
+		padding-bottom: 4px;
+	}
+}
+
+.marked-lines__line-content {
+	width: 100%;
+}
+
+.marked-lines__mark {
+	background-color: #d43a3f;
```

Comment:
To make this component more reusable, this color should be moved to a class that can be used together with the line range. Something like 
```css
.marked-lines__mark {
    &.is-error {
        background-color: #d43a3f;
    }
```

Reply:
what reuse does that buy us? I haven't seen any ned yet of any kind of error mark - just a basic highlighting/indication… am I missing something, like do we know we need to have marks of different colors?

Reply:
Just because it would be ready if we ever want to use it for something else than marking threats. For example, marking in green code additions introduced by a hotfix. The component name MarkedLines leads me to think that I can use it for something else than marking threat lines.

Reply:
Thanks for clarifying. Right now it's not specifically built for threats, but for marks/indicators/highlights. If someone wants to change the color of that
mark they can just by overriding the style

```css
.my-thing .marked-lines__mark {
  background-color: green;
  color: white;
}
```

I'm not opposed to generalizing even more, but I'd rather wait until we
have a need because once we know what's insufficient about it we'll be
able to make a better decision on what it needs.